breathecode/jobs/tests/actions/test_parse_date.py

```diff
@@ -0,0 +1,56 @@
+"""
+Tasks tests
+"""
+from unittest.mock import patch, call
+from ...actions import parse_date
+from ..mixins import JobsTestCase
+from datetime import datetime, timedelta, date
+# from datetime import date
+from breathecode.tests.mocks import (
+    REQUESTS_PATH,
+    apply_requests_post_mock,
+)
+
+
+class ActionRunSpiderTestCase(JobsTestCase):
+    """Tests action certificate_screenshot"""
+
+    # @patch(REQUESTS_PATH['post'],
+    #        apply_requests_post_mock([(200, 'https://app.scrapinghub.com/api/run.json', DATA)]))
+    def test_parse_date__without_job(self):
+        """Test /run_spider without spider"""
+        try:
+            parse_date(None)
+        except Exception as e:
+            self.assertEquals(str(e), ('First you must specify a job'))
+
+    def test_parse_date__verify_format_published_date(self):
+        """Test /run_spider without spider"""
+        # 30+ days ago July 17, 1977 Active 6 days ago
+        model = self.generate_models(job=True, job_kwargs={'published_date_raw': '30+ days ago'})
+
+        result = parse_date(model.job)
+        result = result.published_date_processed
+        result = f'{result.year}-{result.month}-{result.day}'
+        expected = datetime.now() - timedelta(days=30)
+        expected = f'{expected.year}-{expected.month}-{expected.day}'
+
+        self.assertEquals(result, expected)
+
+        model_1 = self.generate_models(job=True, job_kwargs={'published_date_raw': 'Active 6 days ago'})
+
+        result_1 = parse_date(model_1.job)
```

Comment:
Check all the result of this function instead of `f'{expected_1.year}-{expected_1.month}-{expected_1.day}'`